selftests/functional/test_basic.py

```diff
@@ -1034,12 +1072,10 @@ def run_config(config_path):
 
         result_plugins = ["json", "xunit", "zip_archive"]
         result_outputs = ["results.json", "results.xml"]
-        try:
+        if html_uncapable():
```

Comment:
This should now be:

```python
if not html_uncapable():
    result_plugins.append("html")
    result_outputs.append("results.html")
```

Reply:
Oh, sure, I copied the `html_uncapable` from the other test without thinking. I'll change it to `html_capable` in v3.